antlr-kotlin-runtime/build.gradle.kts

```diff
@@ -1,13 +1,20 @@
-import com.strumenta.kotlinmultiplatform.gradle.ext.targetsNative
+import com.strumenta.kotlinmultiplatform.gradle.ext.*
 import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode
+import org.jetbrains.dokka.gradle.DokkaTask
+import com.vanniktech.maven.publish.SonatypeHost
 
 plugins {
   id("strumenta.multiplatform")
+  id("org.jetbrains.dokka")
+  id("com.vanniktech.maven.publish")
 }
 
 strumentaMultiplatform {
   applyJvm()
-  applyJs()
+
+  if (targetsJS()) {
```

Comment:
Is this still necessary?

Reply:
Having the possibility to switch off JS could be useful during development to speed up development, I think

Regarding native, I made the release from home where I did not have yet installed the components to build the native stuff, and given it was just a tentative release I thought I would proceed anyway, but for "real" releases (i.e., releases not marked as RC) I plan to release also for native

Reply:
I installed Xcode and ran again the release task, without excluding native, so release 0.1.0-RC6 should include also the native builds (it should propagate to Maven Central in the next 20-30 minutes)

Reply:
I also updated the Readme to specify that publishing should now include the native targets

Reply:
Sounds good then!

> Having the possibility to switch off JS could be useful during development to speed up development

I'd keep both JVM and JS always on. KMP has potential issues when a single platform is activated (see meta issue https://youtrack.jetbrains.com/issue/KT-52664).

Reply:
We should be sure to catch problems with JS as it would cause the build to fail (this would be evident in PRs and we should get emails too)